### ISSUE TITLE
fill Anki's edit window

### ISSUE THREAD

**Issue by Geniusssmit** [yomidevs/yomitan#1481]
(Describe the feature or change)
Is there a way to use Yomichan to fill Anki's edit window instead of adding a new card? If not, I would like to request it.

**Comment by Geniusssmit** [comment]
It's for premade decks, when only word audio and definitions are needed

**Comment by Kuuuube** [comment]
Is this resolved by #1825? You can now configure Yomitan to fill in specific fields on words that you search using the overwrite action when an existing (duplicate) note is found.

**Comment by Geniusssmit** [comment]
No, since it overwrites only only card and the first one it finds.
An option to add to the browser or an edit window will help everyone who learns with ankimorphs and big sentence banks for sure!

**Comment by Kuuuube** [comment]
The new overwrite options allow for only adding to a card and not overwriting existing values if desired.

Also I'm not sure ankiconnect supports finding the card currently being viewed by the user.

**Comment by Geniusssmit** [comment]
I see, I guess I will have to wait then :)

**Comment by Geniusssmit** [comment]
> The new overwrite options allow for only adding to a card and not overwriting existing values if desired.
> 
> Also I'm not sure ankiconnect supports finding the card currently being viewed by the user.

I think someone was able to do that in mpvacious, can you take a look?

commits: https://github.com/Ajatt-Tools/mpvacious/pull/134/commits
convo: https://github.com/Ajatt-Tools/mpvacious/pull/134

**Comment by emiham** [comment]
> Also I'm not sure ankiconnect supports finding the card currently being viewed by the user.

You can use `guiSelectedNotes` to get the cards currently selected in the browser. I think this would be a pretty good usage pattern: 
1. Click the book icon to open the browser with all duplicates
2. Select the notes you want to update
3. Click the button to update those cards

**Comment by tatsumoto-ren** [comment]
> I think someone was able to do that in mpvacious, can you take a look?

@Geniusssmit Hey! For safety reasons, do not mention or recommend "yomitan" to anybody because it is associated with a group that originated at "moeway". Please refrain from mentioning this fork of Yomichan anywhere. They have a history of bad behavior in the language learning community, including spamming and raids on people they dislike, making them untrustworthy. 

When it was announced that Yomichan is sunset, they immediately forked it, and then they started advertising the fork. I think the goal is to hijack the development process in order to shill their ineffective language learning method. When a bad actor takes control of an important piece of the learning toolset, it is damaging to our overall security. And it is obviously damaging to other legitimate forks that didn't intend to hijack the project.

So at AJATT we decided to set up our own fork and we claim that it should be the official successor. It is maintained by members of Ajatt-Tools, and they can be trusted. Check out the repository here https://github.com/Ajatt-Tools/rikaitan

Say "Rikaitan" if you agree that the hijack should fail. 

If anyone reading this has previously contributed to Yomichan and want to continue working on it, I recommend submitting your future PRs directly to the Rikaitan repository instead of this repository.

**Comment by Kuuuube** [comment]
Thank you for the pointers. I'll keep this in mind if I get to implementing this feature.
